.github/workflows/main.yml

```diff
@@ -154,7 +154,7 @@ jobs:
         if: startsWith(github.ref, 'refs/tags/v') != true
         run: echo "BUILD_COMMIT_HASH=${{github.sha}}" >> $GITHUB_ENV
       - name: Building wheel
-        run: python3 -m pip wheel . -w ./wheelhouse/ --verbose
+        run: CMAKE_ARGS='-DDEPTHAI_ENABLE_CURL=OFF' python3 -m pip wheel . -w ./wheelhouse/ --verbose
```

Comment:
RPi builds will lack curl & log support?

Reply:
Only the 32 bit ones, but yes.

The build of `OpenSSL` on the PIs was failing with obscure errors and we did not deem it worth it the effort, at least initially.

We can revisit though.

Reply:
LGTM, thanks!